auth0/src/main/java/com/auth0/android/request/internal/OkHttpClientFactory.java

```diff
@@ -14,32 +17,49 @@
 
 import javax.net.ssl.SSLContext;
 
-public class OkHttpTLS12Compat {
+public class OkHttpClientFactory {
```

Comment:
Please add an additional javadoc at the class level:


> Factory class used to configure and obtain a new OkHttpClient instance. This class is meant for internal use only, breaking changes may appear at any time without backwards compatibility guarantee. 

or something like that 🙂